diff --git a/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js b/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js
index 7f225f4..ea270ba 100644
--- a/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js
+++ b/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js
@@ -13,8 +13,9 @@ function Address({
   }
   return /*#__PURE__*/jsx("span", {
     "data-testid": "ockAddress",
+    "id": "curAddress",
     className: cn(text.label2, className),
-    children: getSlicedAddress(contextAddress ?? address)
+    children: contextAddress ?? address
   });
 }
 export { Address };
diff --git a/node_modules/@coinbase/onchainkit/esm/wallet/components/Wallet.js b/node_modules/@coinbase/onchainkit/esm/wallet/components/Wallet.js
index 6d25cc6..53abc92 100644
--- a/node_modules/@coinbase/onchainkit/esm/wallet/components/Wallet.js
+++ b/node_modules/@coinbase/onchainkit/esm/wallet/components/Wallet.js
@@ -23,7 +23,7 @@ function Wallet({
     dropdown = _useMemo.dropdown;
   return /*#__PURE__*/jsx(WalletProvider, {
     children: /*#__PURE__*/jsxs("div", {
-      className: "relative w-fit shrink-0",
+      className: "inline-flex relative w-fit shrink-0",
       children: [connect, dropdown]
     })
   });
