diff --git a/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js b/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js
index 7f225f4..ea270ba 100644
--- a/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js
+++ b/node_modules/@coinbase/onchainkit/esm/identity/components/Address.js
@@ -13,8 +13,9 @@ function Address({
   }
   return /*#__PURE__*/jsx("span", {
     "data-testid": "ockAddress",
+    "id": "curAddress",
     className: cn(text.label2, className),
-    children: getSlicedAddress(contextAddress ?? address)
+    children: contextAddress ?? address
   });
 }
 export { Address };
